src/controllers/badgeclassController.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from "express";
+const badgeclass = require('../models/badgeclass');
+const tools = require('../bin/tools');
+
+//TODO: any -> correct type
```

Comment:
Exactly :). Badge classes, assertions and badges are probably prime candidates for custom types